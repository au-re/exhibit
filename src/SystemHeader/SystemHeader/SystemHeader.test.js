import { SystemHeader } from "../../lib";
import React from "react";
import { shallow } from "enzyme";

it("renders without parameters", () => {
  expect(shallow(<SystemHeader />)).toMatchSnapshot();
});

it("renders with children", () => {
  expect(shallow(<SystemHeader><div>test</div></SystemHeader>)).toMatchSnapshot();
});
