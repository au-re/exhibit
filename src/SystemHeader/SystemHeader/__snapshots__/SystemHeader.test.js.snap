// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with children 1`] = `
<div
  className="ReactExhibit__Header"
>
  <div
    className="Header__Banner"
  >
    <div>
      test
    </div>
  </div>
</div>
`;

exports[`renders without parameters 1`] = `
<div
  className="ReactExhibit__Header"
>
  <div
    className="Header__Banner"
  />
</div>
`;
